app/controllers/presentations_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PresentationsController < ApplicationController
   def index
-    @presentations = Presentation.page(params[:page]).order(:created_at)
+    @presentations = Presentation.page(params[:page]).order(created_at: :desc)
```

Comment:
What if I'm backlogging presentations into the system? Should those also appear at the top? `created_at` is useful when the thing you're modelling doesn't have any other timestamps on it. Because our items do have an associated date with them I would expect them to use that for sorting.

I would prefer to see it use the event date for sorting.